### DISCUSSION TITLE
建议增加杜比视界清晰度下载

### DISCUSSION THREAD

**Discussion by lns103** [the1812/Bilibili-Evolved#2161] [功能建议 / Ideas]
B站与近日正式开放了杜比视界规格的上传支持，但是只有支持的手机端才能观看相应画质，有没有可能下载杜比视界的视频来本地观看?
支持的视频：[BV1oq4y1S7T1](https://www.bilibili.com/video/BV1oq4y1S7T1)

**Comment by Geeyun-JY3** [comment]


**Reply by lns103** [reply]
杜比视界是今天才开放上传的啊，和杜比全景声不是一个东西，杜比视界是视频格式，杜比全景声是音频格式
官方介绍链接：[https://b23.tv/cOSk1H](https://b23.tv/cOSk1H) 目前只能在手机上开启

**Reply by Geeyun-JY3** [reply]
啊抱歉看走眼了。
当我没说。

**Comment by lns103** [comment]
分享一下Windows 10 pc 观看杜比视界的插件
链接：https://pan.baidu.com/s/1a7-wi9aSfmZSjrbwUEgWvw 
提取码：1234
这个插件可以把杜比视界转为HDR或者SDR播放

**Reply by lns103** [reply]
支持保留杜比视界的音视频合并命令 ```ffmpeg -i video.m4s -i audio.m4s -strict unofficial -c copy output.mp4```
主要是要保留视频流的 Side data : ```DOVI configuration record: version: 1.0, profile: 8, level: 9, rpu flag: 1, el flag: 0, bl flag: 1, compatibility id: 4```
